### ISSUE TITLE
0.101.1 :Cannot load tensor exported by exportsd.py from torchsharp

### ISSUE THREAD

**Issue by williamlzw** [dotnet/TorchSharp#1144]
python
        
        def test_save_tensor():
            latents = torch.load("latents9.pt")
            exportsd.write_tensor(latents.detach(), "latents.dat")
        
        c#
        var image = torch.load("latents.dat");//System.Runtime.InteropServices.ExternalException:“PytorchStreamReader failed reading zip archive: failed finding central directory

**Comment by NiklasGustafsson** [comment]
Well, I haven't released a version that loads individual tensors, yet. You filed that bug last week, and I have it merged, but the most recent NuGet release doesn't have it.

**Comment by NiklasGustafsson** [comment]
v0.101.2 has been released. Please not that loading tensors in TorchSharp looks like this:

```C#
torch.Tensor x = Tensor.Load("latents");
```

**Comment by williamlzw** [comment]
Great.
